vars/rhel7-dotnet21-container.yml

```diff
@@ -1,6 +1,6 @@
 registry_redhat_io: "dotnet/dotnet-21-rhel7"
-tag_name: "2.1"
-deployment: "oc new-app dotnet:2.1~https://github.com/redhat-developer/s2i-dotnetcore-ex#dotnetcore-2.1 --context-dir=app"
+tag_name: "dotnet21"
+deployment: "oc new-app dotnet21:latest~https://github.com/redhat-developer/s2i-dotnetcore-ex#dotnetcore-2.1 --context-dir=app"
```

Comment:
This should be `oc import-image dotnet/dotnet-21-rhel7 --from=registry.redhat.io/dotnet/dotnet-21-rhel7 --confirm` based on the information from https://catalog.redhat.com/software/containers/dotnet/dotnet-21-rhel7/5b1e83bf4841f106621a7336

Reply:
The same bellow for version 3.1

Reply:
Usage of the `--from` flag with an imagestream indicates that the name entered is just an arbitrary name for local use. Regardless of what name is used in the command below, it can be seen that it's the same image with `oc get istag`.

```
oc import-image my-local-name --from=registry.redhat.io/dotnet/dotnet-21-rhel7 --confirm
```

Reply:
Barring that, I believe that the requested changes are not necessary.